environment.yml

```diff
@@ -0,0 +1,203 @@
+name: ankinator
```

Comment:
Could you only export explicitly installed dependencies, otherwise this environment will not work for everyone `(conda env export --from-history)`

Reply:
doesn't work, because the pip dependencies are missing when doing it this way. Second pip only export has been added to ensure compatibility.